components/manifest-editor/src/components/manifest-screenshots-form.ts

```diff
@@ -265,6 +265,12 @@ export class ManifestScreenshotsForm extends LitElement {
       let passed = validation!.valid;
 
       if(!passed){
+
+        if(this.shadowRoot!.querySelector(`.error-message`)){
+          let error_p = this.shadowRoot!.querySelector(`.error-message`);
```

Comment:
I dont think we need to change it for this PR, but we should try to ensure we use lit's properties and state mechanisms to handle DOM changes. We should keep manual DOM manipulation to a minimum if possible.